src/styles/default.css

```diff
@@ -101,23 +101,23 @@ footer {
 }
 
 .header-item {
-  @apply px-2 py-2 text-zinc-hot-pink bg-zinc-dark-purple border-t-2 border-l-2 border-b-2 border-zinc-hot-pink;
+  @apply px-2 py-2 text-zinc-pastel-pink bg-zinc-dark-purple border-t-2 border-l-2 border-r-2 border-b-2 border-zinc-pastel-pink drop-shadow-xl;
 }
 
 .header-item:last-child {
-  @apply border-r-2 border-zinc-hot-pink;
+  @apply border-r-2 border-zinc-pastel-pink;
 }
 
 .header-item:hover,
 .header-item:active {
-  @apply bg-zinc-dark-purple text-zinc-hot-pink;
+  @apply bg-zinc-dark-purple text-zinc-teal-blue;
 }
 
 .dropdown-container {
   visibility: hidden;
   display: none;
   opacity: 0;
-  @apply absolute bg-zinc-dark-purple z-10;
+  @apply absolute bg-zinc-dark-purple z-10 rounded-lg;
```

Comment:
Need help with alignment.
![help](https://github.com/zinc-collective/www.zinc.coop/assets/16140873/149edee5-9a0b-4c57-b44e-fab6c9e976ee)

Reply:
My changes also somehow changed the width of the sub menu. Does anyone prefer the narrower sub menu over the original sub menu width?

Reply:
In the definition of `dropdown-container`, you can see that it is using `absolute` positioning, with no other modifiers, so it is defaulting to the start of its parent -- the menu text. (See https://tailwindcss.com/docs/position#absolutely-positioning-elements for docs on absolute positioning.)

If you wanted to move this absolutely-positioned sub-menu to the left, I think you need to give it explicit position arguments, in this case with a negative left value, to move the sub-menu to the left by the same amount as the left-padding of the parent. See these docs on negative positioning classes in Tailwind: https://tailwindcss.com/docs/position#using-negative-values

Reply:
I'm not quite sure what is going on with the difference in the width, but it seems like it has started to wrap the text, so maybe adding no-wrap would help (https://tailwindcss.com/docs/whitespace#no-wrap).

Reply:
ty. fixed! 8ed08c28379b2b18a3bc4afddba5e79755aad545